assets/stylesheets/kitten/token/_colors.scss

```
@@ -0,0 +1,69 @@
+:root {
+  // GREY
+  --color-grey-000: #FFFFFF;
+  --color-grey-100: #FBFBFB;
+  --color-grey-200: #F6F6F6;
+  --color-grey-300: #EEEEEE;
+  --color-grey-400: #D8D8D8;
+  --color-grey-500: #B8B8B8;
+  --color-grey-600: #949494;
+  --color-grey-700: #636363;
+  --color-grey-800: #2D2D2D;
+  --color-grey-900: #222222;
+
+  // RED
+  --color-red-100: #FFE9EF;
+  --color-red-300: #FFB8CB;
+  --color-red-500: #F60044;
+  --color-red-700: #D8043E;
+  --color-red-900: #A8002E;
+
+  // GREEN
+  --color-green-100: #E4FBF4;
+  --color-green-300: #A4E7D3;
+  --color-green-500: #08A777;
+  --color-green-700: #0E7758;
+  --color-green-900: #004933;
+
+  // BLUE
+  --color-blue-100: #E7F1FF;
+  --color-blue-300: #7BB3FF;
+  --color-blue-500: #006CFF;
+  --color-blue-700: #0050BD;
+  --color-blue-900: #003A88;
+
+  // YELLOW
+  --color-yellow-100: #FFF3D2;
+  --color-yellow-300: #FFDD84;
+  --color-yellow-500: #FFB800;
+  --color-yellow-700: #A47600;
+  --color-yellow-900: #4E3800;
+
+  // PRIMARY
+  --color-primary-100: #e8f7fe; // old value, use var(--color-blue-100);
+  --color-primary-300: #bae8fd; // old value, use var(--color-blue-300);
+  --color-primary-500: #19b4fa; // old value, use var(--color-blue-500);
+  --color-primary-700: #05a8e6; // old value, use var(--color-blue-700);
+  --color-primary-900: #0496cc; // old value, use var(--color-blue-900);
+
+  // DANGER
+  --color-danger-100: #ffe5ec; // old value, use var(--color-danger-100);
```

Ici, tu parles bien des nouvelles variables à utiliser ?
```suggestion
  --color-danger-100: #ffe5ec; // old value, use var(--color-red-100);
```

---

Bien vu, merci !